ui/Dockerfile.dev

```diff
@@ -1,14 +1,14 @@
 # Note: This is the dockerfile for development purposes
 
 # We are aware that some dev dependencies are broken in node 14, for example chokidar
-# npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. 
+# npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+.
```

Comment:
Unrelated to this PR, but this comment can also be removed now. Chokidar was upgraded by the UI work.